docs/src/api.md

```diff
@@ -233,6 +233,8 @@ read_phantom_MRiLab
 CurrentModule = KomaMRICore
 ```
 
+**KomaMRICore** has **Functors.jl** as one of its dependencies. Functors allow us to easily handle large or nested structures, enabling efficient operations on all their parameters simultaneously, such as changing the precision of values. In **KomaMRI**, four structures are treated as functors internally using the `@functor` macro: `Phantom`, `DiscreteSequence`, `Spinnor`, and `Mag`.
+
```

Comment:
```suggestion
**KomaMRICore** uses the Julia packages **Functors.jl** and **Adapt.jl**, which allows us to easily convert large nested structures. This could be used to send a custom type to the GPU, back to the CPU, change its floating point precision, etc. Four custom types in Koma implement these features: `Phantom`, `DiscreteSequence`, `Spinor`, and `Mag`. More information about these custom types is presented below.

```

Please ensure you have not misspelled `Spinor` as `Spinnor` elsewhere.